json_serializable/test/config_test.dart

```diff
@@ -36,6 +36,11 @@ void main() {
     expect(builder, isNotNull);
   });
 
+  test('triggers config is ignored', () async {
+    // This key is used by build_runner so it's meaningful for any builder.
+    jsonSerializable(const BuilderOptions({'run_only_if_triggered': true}));
```

Comment:
Uh...is any result measured or anything?

Reply:
Nope ... before this PR it throws, so doing nothing is an improvement ;)

I can call that out more explicitly if you like? :)